com.jetbrains.CLion.yml

```diff
@@ -0,0 +1,65 @@
+app-id: com.jetbrains.CLion
+command: clion-desktop
+runtime: org.freedesktop.Sdk
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+finish-args: 
+  - --device=all
+  - --filesystem=home
+  - --filesystem=xdg-run/keyring
+  - --share=ipc
+  - --share=network
+  - --socket=ssh-auth
+  - --socket=wayland
+  - --socket=x11
+  - --talk-name=org.freedesktop.Flatpak
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+add-extensions:
+  org.freedesktop.Sdk.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: "20.08"
+
+modules:   
+  - udev-175.json
```

Comment:
What is it needed for?

Reply:
Embedded development with CLion. I actually just need libudev but it looks like it was added to org.freedesktop.Platform at some point. I can try dropping it and testing it out.

Reply:
I tested it out w/o udev module and the embedded use case works fine. Thanks for catching that one.